BlazarUI/app/scripts/components/build/Build.jsx

```diff
@@ -60,6 +60,7 @@ class Build extends Component {
           <UIGridItem size={12}>
             <BuildLog
               log={this.props.log}
+              isBuilding={build.state === 'IN_PROGRESS'}
```

Comment:
We should make a `BuildStates` module that enumerates all the build states. Then you can do something like:

```javascript
// /constats/BuildStates.js
export default keyMirror({
  IN_PROGRESS: null,
  // etc

// Build.jsx
// ... snip ...
    isBuilding={build.state === BuildStates.IN_PROGRESS}
// ... snip ...
});
```

It's nice to have a central place with all build states within the context of the UI code instead of having to find all `build.state` usages to grok all possible values.

More benefits [here](http://stackoverflow.com/a/31347918)